gui/default/syncthing/core/localeService.js

```diff
@@ -78,11 +78,16 @@ angular.module('syncthing.core')
                                 // case. We compare to the lowercase version of the language
```

Comment:
The `/rest/langs` in the comment above, I had changed that on purpose.  It was renamed at some point and now lives at `/svc/lang`.

*GitHub wouldn't let me add a suggestion to that line though.*

Reply:
Done.